ddtrace/contrib/langchain/patch.py

```diff
@@ -946,6 +946,14 @@ def traced_similarity_search(langchain, pin, func, instance, args, kwargs):
         integration.metric(span, "incr", "request.error", 1)
         raise
     finally:
+        if integration.is_pc_sampled_llmobs(span):
```

Comment:
Based on the test failures, it looks like this span is not getting submitted to LLMObs at all. All `integration.llmobs_set_tags()` does is enhance the APM span with temporary tags.
Take a look at https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/llmobs/_integrations/base.py#L113 to see if there's any options we need to set to mark this span to be submitted to LLMObs.